package.json

```diff
@@ -214,7 +215,12 @@
               },
               "target": {
                 "type": "string",
-                "description": "'simulator', 'device', or the name of the emulator to run on"
+                "description": "Name of the emulator or device to run on"
+              },
+              "targetType": {
+                "type": "string",
+                "description": "Type of target property ('simulator' or 'device')",
```

Comment:
You might want leverage `"enum"` property to define avaiable values

![image](https://user-images.githubusercontent.com/3857604/29555604-df1887a8-872b-11e7-9971-a6169f278a5b.png)

See https://spacetelescope.github.io/understanding-json-schema/reference/generic.html#enumerated-values